From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 29 Oct 2018 01:23:42 +0000
Subject: x86/fpu: Avoid ABI change in 4.9.133
Forwarded: not-needed

Commit 60696d91bbd6 "x86/fpu: Remove struct fpu::counter" did just
what it said.  This is not a real ABI change as that member was
private and is a single byte that becomes a padding byte.  However
genksyms thinks otherwise - and it's just possible that later changes
might cause a real size change.  So add the member back, but mark it
clearly as padding.

---
--- a/arch/x86/include/asm/fpu/types.h
+++ b/arch/x86/include/asm/fpu/types.h
@@ -321,6 +321,12 @@ struct fpu {
 	 */
 	unsigned char			fpregs_active;
 
+#ifndef __GENKSYMS__
+	unsigned char                   __pad_was_counter;
+#else
+	unsigned char                   counter;
+#endif
+
 	/*
 	 * @state:
 	 *
